src/components/InfiniteScrollPaginator/ReverseInfiniteScroll.js

```diff
@@ -224,6 +224,8 @@ class ReverseInfiniteScroll extends Component {
       this.scrollComponent = node;
     };
 
+    props['data-testid'] = 'message-list';
```

Comment:
this could be confusing since we also have a MessageList component

Reply:
Oh yeah, sorry!! That needs to be fixed ... I forgot to fix it. THanks!!